Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -397,6 +400,9 @@ public struct TargetSourcesBuilder {
                 }
             }
 
+            // Ignore README files.
+            if path.basename.hasPrefix("README") { continue}
```

Comment:
Alternatively, to also accommodate variations like `readme-en.md`

```suggestion
            if path.basename.uppercased().hasPrefix("README") { continue }
```